projects/views.py

```diff
@@ -0,0 +1,306 @@
+from django.views.generic import ListView, DetailView, CreateView, UpdateView, DeleteView
+from .models import *
+from django.http import HttpResponseRedirect
+from datetime import datetime
+import reversion
```

Comment:
The order of imports should be (according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)):

> standard library imports
> related third party imports
> local application/library specific imports
> 
> You should put a blank line between each group of imports.